src/materials/ConcreteThermalMoisture.C

```diff
@@ -27,57 +27,50 @@ ConcreteThermalMoisture::validParams()
   params.addRequiredParam<std::string>(
       "type", "A string representing the Moose Object that is used to call this class");
 
-  // parameters for thermal properties calculations
-  MooseEnum thermal_conductivity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum thermal_capacity_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
-  MooseEnum aggregate_type("Siliceous Carbonate", "Siliceous");
-
+  // available model for thermal and moisture transport
+  MooseEnum thermal_transport_model("CONSTANT ASCE-1992 KODUR-2004 EUROCODE-2004", "CONSTANT");
   params.addParam<MooseEnum>(
-      "thermal_conductivity_model", thermal_conductivity_model, "thermal conductivity models");
+      "thermal_transport_model", thermal_transport_model, "thermal conductivity models");
+  MooseEnum moisture_transport_model("Bazant Mensi Xi", "Bazant");
   params.addParam<MooseEnum>(
-      "thermal_capacity_model", thermal_capacity_model, "thermal capacity models");
-  params.addParam<MooseEnum>("aggregate_type", aggregate_type, "siliceous or carbonate");
-
-  params.addParam<Real>(
-      "ref_density_of_concrete", 2231.0, "refernece density of porous media Kg/m^3");
-  params.addParam<Real>(
-      "ref_specific_heat_of_concrete", 1100.0, "reference specific heat of concrete J/Kg/0C");
-  params.addParam<Real>("ref_thermal_conductivity_of_concrete",
-                        3.0,
-                        "reference thermal conductivity of concrete  W/m/0C");
+      "moisture_transport_model", moisture_transport_model, "moisture diffusivity models");
```

Comment:
I'd suggest these changes:

Change parameter name 'moisture_transport_model' to 'moisture_model'. I could be convinced to keep it how you have it if you prefer that, though.
Change parameter description to: 'Model for properties used in moisture transport'